workers/wfsfeature-harvester/src/main/java/org/fao/geonet/harvester/wfsfeatures/worker/EsWFSFeatureIndexer.java

```diff
@@ -260,7 +256,10 @@ public void indexFeatures(Exchange exchange) throws Exception {
             BulkResutHandler brh = new AsyncBulkResutHandler(phaser, typeName, url, nbOfFeatures, report);
 
             long begin = System.currentTimeMillis();
-            FeatureIterator<SimpleFeature> features = wfs.getFeatureSource(typeName).getFeatures(query).features();
+
+            ReprojectingFeatureCollection rfc = new ReprojectingFeatureCollection(wfs.getFeatureSource(typeName).getFeatures(query), CRS.decode("urn:ogc:def:crs:OGC:1.3:CRS84"));
+
+            FeatureIterator<SimpleFeature> features = rfc.features();
```

Comment:
Use try/finally block to ensure FeatureIterator is closed.

Reply:
Done